src/Pos/Wallet/Web/Api.hs

```diff
@@ -77,52 +77,65 @@ type TestReset = API
     :> "reset"
     :> Post '[JSON] (Either WalletError ())
 
+-------------------------------------------------------------------------
+-- Wallet sets
+-------------------------------------------------------------------------
 
 type GetWalletSet = API
-    :> "walletSets"
+    :> "wallets"
```

Comment:
To be honest - I likeed `walletSet`. A second point is that we don't have to have REST api here at all any more as communication between Daedalus and backend will be over TLS, with a specific CA - so the api won't be used "by humans" long term. The idea of client-api is to have a library that talks to the backend